test/updateIn-spec.js

```diff
@@ -27,6 +27,12 @@ describe('u.updateIn', () => {
     expect(result).to.eql({ a: [0, 3, 0] });
   });
 
+  it('can create array if all keys are numbers', () => {
+    const result = u.updateIn('a.1', 3, null);
+
+    expect(result).to.eql({ a: [, 3] });
```

Comment:
This leaves the linter unhappy, but I am not sure how to test this. Creating a new array and setting a value at an index `i` where `i` is greater than array.length results in some an array with some elements missing [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#Relationship_between_length_and_numerical_properties). This seems fine from an application level, but testing it in this way makes the linter unhappy. 

Should I just disable the linter warnings for this line?